src/components/reference/_fork_existing_rule.md

```diff
@@ -0,0 +1,20 @@
+One way to create new rules is to fork an existing rule in Semgrep Registry and modify it to meet your software and business requirements.
```

Comment:
```suggestion
One way to create new rules is to fork an existing rule in the Semgrep Registry and modify it to meet your software and business requirements.
```

These are very murky waters but the Semgrep Registry is not a T1 product so I think we can use articles for it as though it were a common noun.

(For context, T1 products are treated like Proper Nouns even though they might use common nouns; a good example is Semgrep Assistant. For non-Semgrep examples, see the [GitHub Copilot docs](https://docs.github.com/en/copilot/using-github-copilot/getting-code-suggestions-in-your-ide-with-github-copilot) - you'd normally say "the copilot has taken over the flight" in regular English, but no such a/an/the article appears for GitHub's specific branded Copilot.)

Reply:
cc @khorne3 if you have any issues with how English is getting bent out of shape with all the common-noun trademarking in tech.